compiler/souschef/src/Gaussian.cpp

```diff
@@ -26,6 +26,44 @@
 namespace souschef
 {
 
+template <typename T>
+static std::vector<uint8_t> generate_gaussian(int32_t count, float mean, float stddev,
+                                              std::minstd_rand::result_type seed)
+{
+  std::minstd_rand rand{static_cast<std::minstd_rand::result_type>(seed)};
+  std::normal_distribution<float> dist{mean, stddev};
+
+  std::vector<uint8_t> res;
+
+  constexpr float max_cap = std::numeric_limits<T>::max();
+  constexpr float min_cap = std::numeric_limits<T>::min();
+  for (uint32_t n = 0; n < count; ++n)
+  {
+    float raw_value = dist(rand);
+    const float capped_value = std::max(min_cap, std::min(max_cap, raw_value));
```

Comment:
This "capping" is just for reinsurance of correct behavior, standard says: "The behavior is undefined if the truncated value cannot be represented in the destination type." about float -> int conversion.

Though it prbably will work anyway.